src/tactic/doc_commands.lean

```diff
@@ -139,6 +142,11 @@ structure tactic_doc_entry :=
 (description : string := "")
 (inherit_description_from : option _root_.name := none)
 
+instance : inhabited tactic_doc_entry :=
```

Comment:
You can use `@[derive inhabited]` instead (it will produce exactly the same instance).